Documentation/ApiOverview/JavaScript/RequireJS/Shim/Index.rst

```diff
@@ -23,7 +23,7 @@ be defined in the :php:`PageRenderer`::
       [
          'paths' => [
             'jquery' => 'sysext/core/Resources/Public/JavaScript/Contrib/jquery/',
-            'plupload' => '../typo3conf/ext/your_extension/node_modules/plupload/js/plupload.full.min',
+            'plupload' => '/typo3conf/ext/your_extension/node_modules/plupload/js/plupload.full.min',
```

Comment:
```suggestion
            'plupload' => \TYPO3\CMS\Core\Utility\PathUtility::getPublicResourceWebPath('EXT:your_extension/node_modules/plupload/js/plupload.full.min'),
```

Reply:
This is what TYPO3 itself does so we should suggest this as safe and flexible solution.

Reply:
agreed. going to test it now with my case, just in case

Reply:
it works perfectly